### PR TITLE
API 800,1000 and 1200 support for StorageVolumes, StorageVolumeTemplates, StorageSystems, StoragePool

### PR DESCRIPTION
**Pull request by chebroluharika**
**Description**
API 800,1000 and 1200 support for StorageVolumes, StorageVolumeTemplates, StorageSystems, StoragePool

**Issues Resolved**
NA

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by chebroluharika** [comment]
I somehow messed entire PR with unnecessary merges. Hence, I created another PR with same changes. The PR to which I already received few approvals is https://github.com/HewlettPackard/oneview-chef/pull/408

**Review by AnushaY1916 — APPROVED** [review]

**Review by soodpr — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +18,12 @@
 - oneview_server_hardware_type
 - oneview_server_profile
 - oneview_server_profile_template
+- oneview_storage_pool
+- oneview_storage_system
 - oneview_uplink_set
+- oneview_volume
+- oneview_volume_template
+
```
Comment (jyothisgm) [inline CHANGELOG.md:26]: Please remove this extra line.

**Review by jyothisgm — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (chebroluharika) [reply CHANGELOG.md:26]: done

**Pull request merged**